config.yaml

```diff
@@ -14,6 +14,18 @@ arch:
 options:
   log_level: debug
   config_dir: "/config"
+  variable_start_string: "{{"
```

Comment:
It would be better if these didn't have defaults here, then they wouldn't always be visible in the config (the user would see them if 'Show unused optional configuration options' was selected, but otherwise only the basic options would be shown).